dimager/handlers.py

```diff
@@ -0,0 +1,11 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.contrib.auth.models import User
+from dimager.models import ImagerProfile
+
+
+@receiver(post_save, sender=User)
+def create_profile(sender, instance, *args, **kwargs):
```

Comment:
It's recommended by Django to leave the method signature for *all* signal handlers to be `sender`, `*args`, and `**kwargs`.  In other words, although you know that `instance` is a provided item in the call for the `post_save` signal, you should not rely on that implementation detail.  Better to program defensively like so:

```python
def create_profile(sender, *args, **kwargs):
    if kwargs.get('created'):
        user = kwargs.get('instance')
        if user is not None:
            new_profile = ImagerProfile(user=user)
            new_profile.save()
````